guessing_game.rb

```diff
@@ -0,0 +1,81 @@
+require 'pry-byebug'
+
+class GuessingGame
+
+  attr_accessor :congrats_message, :remaining_guesses, :won
+
+  def initialize(secret_number, num_guesses)
+    @congrats_message = "Yay, you won!"
+    @secret_number = secret_number
+    @remaining_guesses = num_guesses
+    @guessed_numbers = []
+    @won = false
+    @lost = false
+  end
+
+  def has_won?
+    @won
+  end
+
+  def has_lost?
+    @lost
+  end
```

Comment:
you can keep track of the game with one variable. `@status == :won` or `@status == :lost` or `@status == :playing`